### PR TITLE
Add mobile Header and Nav

### PR DESCRIPTION
**Pull request by eduardoboucas**

### CODE REVIEW

File: dadi/frontend/lib/util.js [hunk 1]
```diff
@@ -62,6 +77,13 @@ Style.prototype.add = function (className) {
   return this
 }
 
+Style.prototype.addIf = function (className, condition) {
+  console.log('--> condition_', condition)
```
Comment (mingard) [inline dadi/frontend/lib/util.js]: @eduardoboucas still need this?

Comment (eduardoboucas) [inline dadi/frontend/lib/util.js]: Oops, no! I'll remove tomorrow.

**Comment by eduardoboucas** [comment]
Cheers! Can you please make me document the `Style` class tomorrow? 😅

**Comment by mingard** [comment]
@eduardoboucas of course. Lets work on docs roundup tomorrow

**Comment by mingard** [comment]
@eduardoboucas merged, but just to clear the branch

**Pull request merged**